includes/widgets/text-editor.php

```diff
@@ -126,17 +126,25 @@ protected function register_controls() {
 			]
 		);
 
-		$text_columns = range( 1, 10 );
-		$text_columns = array_combine( $text_columns, $text_columns );
-		$text_columns[''] = esc_html__( 'Default', 'elementor' );
-
 		$this->add_responsive_control(
 			'text_columns',
 			[
 				'label' => esc_html__( 'Columns', 'elementor' ),
 				'type' => Controls_Manager::SELECT,
 				'separator' => 'before',
-				'options' => $text_columns,
+				'options' => [
+					'' => esc_html__( 'Default', 'elementor' ),
+					'1' => '1',
+					'2' => '2',
+					'3' => '3',
+					'4' => '4',
+					'5' => '5',
+					'6' => '6',
+					'7' => '7',
+					'8' => '8',
+					'9' => '9',
+					'10' => '10',
```

Comment:
I'm considering to add translation string for languages like Arabic that have their own numbers.